tests/__init__.py

```
@@ -1966,23 +1966,23 @@
     reset_sequences(db)
 
 
-def load_app_db(db, sql, params=None):
+def load_app_db(db, query, params=None):
     # type: (DatabaseContext, str, Any) -> None
-    db.get_app_db().execute(sql, params)
+    db.get_app_db().execute(sql.text(query), params)
 
 
-def load_fingerprint_db(db, sql, params=None):
+def load_fingerprint_db(db, query, params=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `load_fingerprint_db`

Add return type annotation: `None`

(ANN201)